BUILD.bazel

```diff
@@ -74,8 +74,30 @@ buildifier(
     name = "buildifier"
 )
 
+script_base = "$SRCROOT/tools/instrumentation_helpers"
+
 xchammer_config = xchammer_config(
     targets=["//:xchammer"],
+    target_config={
```

Comment:
This is how it logging is intended to be setup ( internally ), but probably will not end up being in XCHammer in this form